src/generate-from-template.js

```diff
@@ -26,6 +26,18 @@ const getPackageName = require('./utils/get-package-name');
 const getPackageVersion = require('./utils/get-package-version');
 const { getStoredValues, setStoreValues } = require('./utils/storage');
 
+const noop = () => 0;
```

Comment:
The docs state that these methods should return true to indicate that a step can be skipped. Can this noon return `false` to keep a consistent type?

Reply:
done